app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Task;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator as FacadesValidator;
+
+class TaskController extends Controller {
+    public function index() {
+       
+
+        return view('tasks', [
+            'tasks' => Task::search()
+        ]);
+    }
+    public function delete($id){
+         
+            Task::findOrFail($id)->delete();
+
+            return redirect('/');
+        
+    }
+    public function  update(Request $request) {
+
+         
+            $validator = FacadesValidator::make($request->all(), [
+                'name' => 'required|max:20',
+            'name' => 'required|min:2',
```

Comment:
why using validation on name two time 
remove line 29 and move `min:2` to line 28

Reply:
solved